package.json

```diff
@@ -34,7 +34,7 @@
         "@appwrite.io/console": "^0.6.4",
         "@appwrite.io/pink": "~0.26.0",
         "@appwrite.io/pink-icons": "~0.26.0",
-        "@appwrite.io/repo": "github:appwrite/appwrite#1.7.x",
+        "@appwrite.io/repo": "github:appwrite/appwrite#feat-definition-examples",
```

Comment:
lets get this one merged on appwrite and then get this one released. Mark as draft till then.